egs/wsj/s5/steps/cleanup/segment_long_utterances.sh

```diff
@@ -50,15 +50,17 @@ cmd=run.pl
 
 . utils/parse_options.sh
 
-if [ $# -ne 6 ]; then
+if [ $# -ne 5 ] && [ $# -ne 7 ]; then
     cat <<EOF
-Usage: $0 [options] <model-dir> <lang> <data-in> <text-in> <segmented-data-out> <work-dir>
+Usage: $0 [options] <model-dir> <lang> <data-in> [<text-in> <utt2text>] <segmented-data-out> <work-dir>
  e.g.: $0 exp/wsj_tri2b data/lang_nosp data/train_long data/train_long/text data/train_reseg exp/segment_wsj_long_utts_train
 This script performs segmentation of the data in <data-in> and 
 transcript <text-in>, writing the segmented data (with a segments file) to
```

Comment:
this documentation does not read well, or does not agree with the usage line.  Please check carefully.